packages/main/src/themes/base/DayPicker-parameters.css

```diff
@@ -2,18 +2,22 @@
 	--_ui5_daypicker_item_margin: 2px;
 	--_ui5_daypicker_item_border: none;
 	--_ui5_daypicker_item_selected_border_color: var(--sapList_Background);
-	--_ui5_daypicker_item_outline_width: 1px;
-	--_ui5_daypicker_item_outline_offset: 1px;
 	--_ui5_daypicker_daynames_container_height: 2rem;
 	--_ui5_daypicker_weeknumbers_container_padding_top: 2rem;
-	--_ui5_daypicker_item_selected_hover_background_color: var(--sapContent_Selected_Background); /* darken(@sapSelectedColor, 10); */
+	--_ui5_daypicker_item_selected_hover_background_color: var(--sapContent_Selected_Hover_Background);
```

Comment:
If you are not changing the value of ```_ui5_daypicker_item_selected_hover_background_color```, just use ```var(--sapContent_Selected_Hover_Background);``` where needed & delete this line